src/inquirer.js

```diff
@@ -0,0 +1,61 @@
+const inquirer = require('inquirer');
+const timestring = require('timestring');
+
+const questions = [{
+  type: 'confirm',
+  name: 'is_enterprise',
+  message: 'Do you want to use Github Enterprise?'
+},
+{
+  type: 'input',
+  name: 'url',
+  message: 'Enter the GitHub URL',
+  default: 'https://api.github.com',
+  when: function (answers) {
+    return answers.is_enterprise;
+  }
+},
+{
+  type: 'password',
+  name: 'token',
+  message: 'Enter the GitHub access token:',
+  validate: function (username) {
+    if (username)
+      var pass = username.match(/^[a-zA-Z0-9_-]+$/);
+    if (pass) {
+      return true;
+    }
+    return 'Please enter a valid username';
+  }
+},
+{
+  type: 'input',
+  name: 'hours',
+  message: 'Enter the timeframe:',
```

Comment:
perhaps we can add (default 1d) to the prompt message so users know they can skip this if they like